src/main/webapp/app/course/tutorial-groups/tutorial-groups-management/tutorial-groups/tutorial-groups-management/tutorial-groups-export-button.component/tutorial-groups-export-button.component.html

```diff
@@ -0,0 +1,48 @@
+<button id="exportDialogButton" (click)="openExportDialog($event)" [jhiTranslate]="'artemisApp.tutorialGroupExportDialog.buttonHeader'" ngbDropdownItem></button>
+
+<ng-template #exportDialog let-modal>
+    <div class="modal-header">
+        <h4 class="modal-title" [jhiTranslate]="'artemisApp.tutorialGroupExportDialog.header'"></h4>
+        <button type="button" class="btn-close" aria-label="Close" (click)="modal.dismiss('cancel')"></button>
+    </div>
+    <div class="modal-body">
+        <form>
+            <p [jhiTranslate]="'artemisApp.tutorialGroupExportDialog.description'"></p>
+            <div>
+                <div class="form-check" style="margin-bottom: 1rem">
+                    <input class="form-check-input" type="checkbox" (change)="toggleSelectAll()" [checked]="selectAll" />
```

Comment:
Ideally, the label should be assigned to the corresponding input using `id` https://docs.artemis.cit.tum.de/dev/guidelines/client.html#labels

Reply:
done 👍